ci/jenkins/pipelines/skuba-upgrade-from-4.2.Jenkinsfile

```diff
@@ -0,0 +1,133 @@
+def cleanup_cluster=true
+
+pipeline {
+
+    agent { label "${WORKER_LABEL}" }
+    
+    environment {
+        SKUBA_BINPATH = "/home/jenkins/go/bin/skuba"
+        OPENSTACK_OPENRC = credentials('ecp-openrc')
+        GITHUB_TOKEN = credentials('github-token')
+        VMWARE_ENV_FILE = credentials('vmware-env')
+        TERRAFORM_STACK_NAME = "${BUILD_NUMBER}-${JOB_NAME.replaceAll("/","-")}".take(70)
+     }
+
+     stages {
+
+        stage("Build skuba"){
+            steps{
+                sh(script: "git checkout ${INITIAL_VERSION}")
```

Comment:
yikes, isnt there a better way to change branches? I heard using git directly was not recommended although I cant remember why.

Reply:
we can review this in a follow up.